.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 coverage
-
-generators/app/templates/\.DS_Store
+/.idea
+/.DS_Store
```

Comment:
It's okay to remove the `/` in front of those lines, those files/folders should be ignored anywhere, not only at root level.